lib/beaker/options/parser.rb

```diff
@@ -90,7 +90,9 @@ def file_list(paths)
       # @return nil
       # @api public
       def resolve_symlinks!
-        @options[:hosts_file] = File.realpath(@options[:hosts_file]) if @options[:hosts_file]
+        if @options[:hosts_file] && !@options[:hosts_file_generated]
```

Comment:
i'm with you on this one.  will vote on a ticket to move to some singleton state object, or similar